File: translations/codenet/gpt_commented/C2J/gpt-4o/Code/s906878895.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Create a Scanner object to read input
        Scanner scanner = new Scanner(System.in);
        
        // Continuously read pairs of integers until EOF (End of File) is reached
        while (scanner.hasNextInt()) {
            int a = scanner.nextInt();
            int b = scanner.nextInt();
            
            // Calculate the sum of the two integers
            int x = a + b;
            
            // Initialize a counter for the number of digits in the sum
            int w = 0;
            
            // Count the number of digits in the sum (x)
            if (x == 0) {
                w = 1; // Special case for 0
            } else {
                while (x != 0) {
                    // Increment the digit counter
                    w++;
                    // Remove the last digit from x by dividing by 10
                    x /= 10;
                }
            }
            
            // Print the number of digits in the sum
            System.out.println(w);
        }
        
        // Close the scanner
        scanner.close();
    }
}

// <END-OF-CODE>

File: translations/codenet/gpt_commented/C2P/granite/Code/s906878895.py

# Python code to calculate the number of digits in the sum of two integers

# Continuously read pairs of integers until EOF (End of File) is reached
while True:
    try:
        # Read two integers from the user
        a, b = map(int, input().split())
        
        # Calculate the sum of the two integers
        x = a + b
        
        # Initialize a counter for the number of digits in the sum
        w = 0
        
        # Count the number of digits in the sum (x)
        while x > 0:
            # Increment the digit counter
            w += 1
            # Remove the last digit from x by dividing by 10
            x //= 10
        
        # Print the number of digits in the sum
        print(w)
    except EOFError:
        # Break the loop when EOF is reached
        break

# End of Python code
